<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>aboutdlgClass</class>
 <widget class="QDialog" name="aboutdlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>573</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>IN01 Info</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="copyright">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>530</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>copyright</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_documentation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Documentation</string>
   </property>
   <widget class="QPushButton" name="bt_doculink">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>link</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="bt_cancel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_application">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Application</string>
   </property>
   <widget class="QWidget" name="verticalLayout">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>101</width>
      <height>96</height>
     </rect>
    </property>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lb_Version">
       <property name="text">
        <string>version</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_Server">
       <property name="text">
        <string>server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_Database">
       <property name="text">
        <string>database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_branch">
       <property name="text">
        <string>area</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_user">
       <property name="text">
        <string>user</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayout_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>281</width>
      <height>96</height>
     </rect>
    </property>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="version">
       <property name="text">
        <string>version</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="server">
       <property name="text">
        <string>server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="database">
       <property name="text">
        <string>database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="branch">
       <property name="text">
        <string>branch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="user">
       <property name="text">
        <string>user</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="product">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>product</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_DLL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>531</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>Dynamic Link Libraries</string>
   </property>
   <widget class="QTableWidget" name="tw_dll">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column/>
    <column/>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>bt_cancel</tabstop>
  <tabstop>tw_dll</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>bt_cancel</sender>
   <signal>clicked()</signal>
   <receiver>aboutdlgClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>530</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
